src/commonMain/kotlin/au/id/micolous/metrodroid/transit/clipper/ClipperData.kt

```diff
@@ -37,6 +37,8 @@ internal object ClipperData {
     const val AGENCY_BAY_FERRY = 0x1b
 
     val GG_FERRY_ROUTES = mapOf(
+            0x01 to FormattedString.language("San Francisco", "en-US"),
```

Comment:
The operator has a [Sausalito ferry schedule](http://goldengateferry.org/schedules/Sausalito.php), so I think `0x01` should be `Sausalito`.  I don't think we have a good way to represent direction yet; so a comment would be fine for now (if we add this later).

If I recall correctly, you only tap on your Clipper card when boarding the ferries.

`0x04` should also be updated, as [Larkspur is the route name](http://goldengateferry.org/schedules/Larkspur.php).

These look the same as the station IDs, but getting this plumbed through GTFS is probably more tedious than you'd like to spend time on.

The data has been like this for a while. Unfortunately the times I've taken this ferry, someone bought a book of (paper) tickets for me.

Reply:
I tested my Clipper card on the ferries and it yielded 0x01 from Sausalito to San Francisco, and 0x02 from San Francisco to Sausalito. They only made me tag on once each time, though both of tags occurred in San Francisco (I tagged before boarding in SF for the SF => Sausalito direction, but tagged after leaving in SF for Sausalito => SF).

The way I think GGF represents the values is a bit unique. All of the routes have only one stop, so the "Route" is the final destination, start is the starting station, and end is "End of Line".

So for example, 0x01 is "San Francisco", Start: "Sausalito", End: "End of Line".

Reply:
Thanks for the explanation!